src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -61,7 +63,7 @@ protected boolean containsExpectedPageContents() {
     }
 
     public String getCourseId() {
-        return browser.driver.findElement(By.id("courseid")).getAttribute("value");
+        return browser.driver.findElement(By.id(Const.ParamsNames.COURSE_ID)).getAttribute("value");
```

Comment:
Since there is a `@FindBy` defined for `courseid`, I think actually can just refer to that